src/site/content/en/blog/best-practices/index.md

```diff
@@ -0,0 +1,557 @@
+---
+title: Custom Element Best Practices
```

Comment:
Rename this file custom-elements-best-practices.md (as we have a flat file structure over here I think we will need to append the old directory name to the filename so we don' get conflicting names.)